build/simp-environment.spec

```diff
@@ -178,21 +176,14 @@ chown -R ${puppet_user}:${puppet_group} %{prefix}/simp_autofiles
 chgrp ${puppet_group} %{prefix}/environment.conf
```

Comment:
Do we need to chgrp these files to puppet_group?

Reply:
They are just copies, they don't 'need' to be changed.  But it doesn't hurt for it to have the correct group on it.  Would you prefer we not do the work here?

Reply:
will defer that work to another ticket